x-pack/filebeat/module/ibmmq/_meta/kibana/7/dashboard/Filebeat-IBMMQ-Overview.json

```diff
@@ -0,0 +1,814 @@
+{
```

Comment:
This dashboard will probably need an update as soon as we finished the field renaming.